walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,13 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + removeNonAlphabetChar(name.trim());
+  }
+
+  /**
+   * Returns name after removing all characters that are not alphabet letters.
+   */
+  public String removeNonAlphabetChar(String name){
+    return name.replaceAll("[^a-zA-Z]", "");
```

Comment:
having said the above this code doesn't allow you to have two names, eg "Lauren Spooner" would become "LaurenSpooner" so maybe keeping the trim and allowing spaces in here is better?